.github/workflows/release-tag.yml

```diff
@@ -0,0 +1,124 @@
+name: Tag release
+
+on:
+  push:
+    tags:
+      - 'v3.[0-9]+.[0-9]+'
+
+env:
+  NODE_ENV: production
+
+jobs:
+  documentation:
+    name: Documentation
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Check out documentation
+      uses: actions/checkout@v2
+      with:
+        ref: gh-pages
+        path: doc/public
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '~1.14'
+    - name: Download Go tool dependencies
+      run: |
+        cd tools
+        go mod download
+    - name: Make Mage
+      run: make tools/bin/mage
+    - name: Install JS SDK dependencies
+      run: tools/bin/mage jsSDK:deps
+    - name: Build JS SDK
+      run: tools/bin/mage jsSDK:clean jsSDK:build
+    - name: Install JS dependencies
+      run: tools/bin/mage js:deps
+    - name: File versioning
+      run: tools/bin/mage version:files
+    - name: Check for diff
+      run: tools/bin/mage git:diff
+    - name: Install documentation dependencies
+      run: tools/bin/mage docs:deps
+    - name: Build documentation
+      run: tools/bin/mage docs:build
+      env:
+        HUGO_BASE_URL: https://thethingsstack.io/
```

Comment:
We also need `HUGO_PARAMS_SEARCH_ENABLED=true`, `HUGO_PARAMS_SEARCH_INDEX=thethingsstack` and `HUGO_PARAMS_SEARCH_APIKEY=<secret>`.